### ISSUE TITLE
[BUG] Schema generates wrong tag for TargetEncoding op

### ISSUE THREAD

**Issue by rnyak** [NVIDIA-Merlin/NVTabular#1359]
**Describe the bug**
The TargetEncoding tag in the `schema.pbtxt` is exported as `categorical` but it should be continuous, since the outputted column values are continuous.

Another issue, if I add `Normalize` op at the end of the TargetEncoding, then we get both `continuous and categorical` tags. A feature cannot have both tags. In this case, I assume the latest op should be considered and previous `tag` should be overwritten.

**Steps/Code to reproduce bug**
Please run the following code to repro the issue:

```
import os
import nvtabular as nvt
from nvtabular import Workflow, ops

gdf = nvt.dispatch._make_df(
    {
        "user_id": [1, 2, 3, 4, 6, 8, 5, 3] *10,
        "rating": [1.5, 2.5, 3.0, 4.0, 5.0, 2.0, 3.0, 1.0] *10
    }
)

cats = ["user_id"]
cat_features = cats >> ops.Categorify()

label = nvt.ColumnSelector(['rating'])

# Target Encode movieId column
te_features = cat_features >> ops.TargetEncoding(
    label, 
    kfold=5, 
    p_smooth=20
)
te_features_norm = te_features >> ops.NormalizeMinMax()

train_dataset = nvt.Dataset(gdf)


workflow = Workflow(te_features_norm)
workflow.fit(train_dataset)

workflow.transform(train_dataset).to_parquet(
    output_path=os.path.join('./', "train"),
    out_files_per_proc=1,
    shuffle=False,
)
```
you should be getting a schema file as below:

```
feature {
  name: "TE_user_id_rating"
  type: FLOAT
  float_domain {
    name: "TE_user_id_rating"
    min: 0.0
    max: 8.0
  }
  annotation {
    tag: "categorical"
    tag: "continuous"
    extra_metadata {
      type_url: "type.googleapis.com/google.protobuf.Struct"
      value: "\n\021\n\013num_buckets\022\002\010\000\n\033\n\016freq_threshold\022\t\021\000\000\000\000\000\000\000\000\n\025\n\010max_size\022\t\021\000\000\000\000\000\000\000\000\n\030\n\013start_index\022\t\021\000\000\000\000\000\000\000\000\n2\n\010cat_path\022&\032$.//categories/unique.user_id.parquet\nG\n\017embedding_sizes\0224*2\n\030\n\013cardinality\022\t\021\000\000\000\000\000\000 @\n\026\n\tdimension\022\t\021\000\000\000\000\000\0000@"
    }
  }
}

````

Note that `TE_user_id_rating` has both `continuous and categorical` tags, which shouldn't have.

**Expected behavior**
TargetEncoding annotation tag in the schema should be `continuous`, and if we chain another op, the schema should consider the latest op for tag type.

**Environment details (please complete the following information):**
 - Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
 - Method of NVTabular install: [conda, Docker, or from source]
   - If method of install is [Docker], provide `docker pull` & `docker run` commands used
 
Using `merlin-tensorflow-training:22.01` container.
